apps/voting/app/src/script.js

```diff
@@ -1,7 +1,7 @@
 import Aragon from '@aragon/client'
 import { of } from './rxjs'
 import voteSettings, { hasLoadedVoteSettings } from './vote-settings'
-import { EMPTY_CALLSCRIPT } from './vote-utils'
+import { EMPTY_CALLSCRIPT } from './evmscript-utils'
```

Comment:
For some reason, parcel doesn't like tree-shaking `vote-utils`, so having `EMPTY_CALLSCRIPT` there results in the bundle being much larger (~150kb minified).